store/storetest/reaction_store.go

```diff
@@ -348,3 +349,69 @@ func testReactionStorePermanentDeleteBatch(t *testing.T, ss store.Store) {
 		t.Fatalf("expected 1 reaction. Got: %v", len(returned))
 	}
 }
+
+func testReactionBulkGetForPosts(t *testing.T, ss store.Store) {
+	postId := model.NewId()
+	post2Id := model.NewId()
+	post3Id := model.NewId()
+	post4Id := model.NewId()
+
+	userId := model.NewId()
+
+	reactions := []*model.Reaction{
+		{
+			UserId:    userId,
+			PostId:    postId,
+			EmojiName: "smile",
+		},
+		{
+			UserId:    model.NewId(),
+			PostId:    post2Id,
+			EmojiName: "smile",
+		},
+		{
+			UserId:    userId,
+			PostId:    post3Id,
+			EmojiName: "sad",
+		},
+		{
+			UserId:    userId,
+			PostId:    postId,
+			EmojiName: "angry",
+		},
+		{
+			UserId:    userId,
+			PostId:    post2Id,
+			EmojiName: "angry",
+		},
+		{
+			UserId:    userId,
+			PostId:    post4Id,
+			EmojiName: "angry",
+		},
+	}
+
+	for _, reaction := range reactions {
+		store.Must(ss.Reaction().Save(reaction))
+	}
+
+	postIds := []string{postId, post2Id, post3Id}
+	if result := <-ss.Reaction().BulkGetForPosts(postIds); result.Err != nil {
+		t.Fatal(result.Err)
+	} else if returned := result.Data.([]*model.Reaction); len(returned) != 5 {
+		t.Fatal("should've returned 5 reactions")
+	} else {
+		post4IdFound := false
+		for _, reaction := range returned {
+			if reaction.PostId == post4Id {
+				post4IdFound = true
+				break
+			}
+		}
+
+		if post4IdFound {
+			t.Fatal("Wrong reactipn returned")
```

Comment:
Minor typo
```suggestion
			t.Fatal("Wrong reaction returned")
```

Also, if you wanted to, you could make this whole section simpler by using `github.com/stretchr/testify/assert`

```go
postIds := []string{postId, post2Id, post3Id}

result := <-ss.Reaction().BulkGetForPosts(postIds)
assert.Nil(t, result.Err)

data := result.Data.([]*model.Reaction)
assert.Len(t, data, 5)
assert.Contains(t, data, post4Id)
```